src/main/java/meteordevelopment/meteorclient/systems/hud/elements/InventoryHud.java

```diff
@@ -35,21 +39,76 @@ public class InventoryHud extends HudElement {
         .build()
     );
 
+    interface SupportedSourcesInterface { ArrayList<ItemStack> getItems(); }
+    private enum SupportedSources implements SupportedSourcesInterface
+    {
+        None {
+            @Override
+            public ArrayList<ItemStack> getItems() { return new ArrayList<>(); }
```

Comment:
`return List.of()`

Reply:
Only works as `return new ArrayList<>(List.of())`.
```
error: incompatible types: no instance(s) of type variable(s) E exist so that List<E> conforms to ArrayList<ItemStack>
```